cohere/client_async.py

```diff
@@ -178,15 +178,19 @@ async def chat(
         return_prompt: bool = False,
         return_preamble: bool = False,
         chatlog_override: List[Dict[str, str]] = None,
+        chat_history: List[Dict[str, str]] = None,
         preamble_override: str = None,
         user_name: str = None,
         temperature: float = 0.8,
-        max_tokens: int = 200,
+        max_tokens: int = None,
         stream: bool = False,
     ) -> Union[AsyncChat, StreamingChat]:
         if chatlog_override is not None:
```

Comment:
shouldn't we add the same docstring? I know the whole setup of duplication is not optimal, but it is what we have atm.

Reply:
I think the docstring is duplicated when rendering [the documentation](https://cohere-sdk.readthedocs.io/en/latest/cohere.html#cohere.client_async.AsyncClient.chat)

Reply:
@lusmoura I was thinking for code intellisense and editor hints. It's ok 👌